packages/Parallel/WorkerRunner.php

```diff
@@ -95,7 +95,7 @@ public function run(Encoder $encoder, Decoder $decoder, Configuration $configura
 
                     if ($errorAndFileDiffs[Bridge::SYSTEM_ERRORS] !== []) {
                         $this->invalidateFile($file);
-                    } elseif (! $configuration->isDryRun()) {
+                    } else if (! $configuration->isDryRun() || $errorAndFileDiffs[Bridge::FILE_DIFFS] === []) {
```

Comment:
WorkerRunner only apply on parallel process, the non-parallel can happen in ApplicationFileProcessor
https://github.com/rectorphp/rector-src/blob/0d6800eea6b74352bd2dd16d7a2254c9398e4e98/src/Application/ApplicationFileProcessor.php#L136

Reply:
Thanks, I will update the PR to reflect this :+1:

Reply:
`else if` can be `elseif` per coding standard we use.